opennms-services/src/main/java/org/opennms/netmgt/snmpinterfacepoller/pollable/PollableSnmpInterface.java

```diff
@@ -284,12 +283,16 @@ public boolean isReady() {
     @Override
     public void run() {
             if (getParent().polling()) {
-                LOG.info("run: polling SNMP interfaces on package/interface {}/{} on primary address: {}", getParent().getPackageName(), getName(), getParent().getIpaddress());
+                String location = getContext().getLocation(getParent().getNodeid());
```

Comment:
We could also store the location in the `PollableInterface` class along with the node ID instead of looking this up every time. Other components in the code assume that a node's location doesn't change throughout it's lifetime, so we don't have to worry about the location changing from one run to another.